fulfil_client/serialization.py

```diff
@@ -46,6 +46,9 @@ def __call__(self, dct):
 JSONDecoder.register(
     'Decimal', lambda dct: Decimal(dct['decimal'])
 )
+JSONDecoder.register(
+    'Model', lambda dct: dct['id']
```

Comment:
I was thinking if someway this can be decoded to Record object but that needs client, may be on flask fulfil. What are your thoughts? Otherwise I will go with `<model>,<id>`?